package.json

```diff
@@ -38,9 +38,9 @@
     ]
   },
   "scripts": {
-    "build": "yarn clean && yarn build:scss && yarn build:css && yarn build:html && yarn copy-files",
+    "build": "yarn clean && yarn build:scss && yarn build:css && yarn lint:copyright --fix lib/css/* && yarn build:html && yarn copy-files",
     "build:scss": "cpx \"./src/**/*.scss\" ./lib/scss",
-    "build:css": "node ./scripts/generateCss.js src lib/css && yarn postcss-processing && yarn lint:copyright --fix lib/css/*",
+    "build:css": "node ./scripts/generateCss.js src lib/css",
```

Comment:
Shouldn't we use both `sass` and `postcss` with their built-in `watch` flag?

Reply:
Our file structure doesn't allow us to do so.

Reply:
I think we could use postcss at least. Because the files are already compiled by that time

Reply:
I could try but I guess the performance would be equal or even worse because CSS needs to be written to a file, then PostCSS needs to read that file and write again.

Reply:
Apparently I can't do that 😀
![image](https://user-images.githubusercontent.com/36186912/158197054-4e751f3c-4d40-4fc3-8b94-c74818c28737.png)
Will leave it for the future.